packages/docusaurus-plugin-content-docs/src/sidebars.ts

```
@@ -210,6 +227,7 @@ function normalizeItem(item: SidebarItemJSON): UnprocessedSidebarItem[] {
       return [
         {
           collapsed: DefaultCategoryCollapsedValue,
+          collapsible: DefaultCategoryCollapsibleValue,
```

replace plugin option default value?

---

also, we could throw an error if `!collapsible && collapsed` as the user probably made a mistake